pom.xml

```diff
@@ -94,7 +91,27 @@
             <artifactId>springdoc-openapi-ui</artifactId>
             <version>1.6.0</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.17.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>1.17.2</version>
```

Comment:
It's better to introduce a property to hold this version and use the property across all testcontainers dependencies

Reply:
Added